### ISSUE TITLE
undefined local variable or method `extra_pages'

### ISSUE THREAD

**Issue by kimadactyl** [devnacho/mountain_view#56]
Heya, 

Just tried to install this following the directions to my existing Rails site. I made a header element with some static info in to test it. When I try and look at the style guide I get 
```
undefined local variable or method `extra_pages' for #<#<Class:0x0055d5763a1ff0>:0x0055d576392fc8> Did you mean?  extract_dimensions.
```

The stacktrace is showing this as being with Spree (I don't see how it could be) so I'm not sure if that is the problem or not, leaving all the output in for clarity, apols if that is the case. I have a lot going on in my app so not sure where to start. 

```
kim@haraway:~/git/taichi/taichi_rails$ bundle exec rails server webrick
=> Booting WEBrick
=> Rails 4.2.9 application starting in development on http://0.0.0.0:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
[2017-08-07 16:26:17] INFO  WEBrick 1.3.1
[2017-08-07 16:26:17] INFO  ruby 2.4.1 (2017-03-22) [x86_64-linux]
[2017-08-07 16:26:17] INFO  WEBrick::HTTPServer#start: pid=20140 port=3000


Started GET "/mountain_view/styleguide" for 127.0.0.1 at 2017-08-07 16:26:32 +0100
  ActiveRecord::SchemaMigration Load (0.6ms)  SELECT "schema_migrations".* FROM "schema_migrations"
Processing by MountainView::StyleguideController#index as HTML
/home/kim/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/versioncake-2.3.1/lib/versioncake/strategies/extraction_strategy.rb:8: warning: constant ::Fixnum is deprecated
  Notification Load (0.2ms)  SELECT  "notifications".* FROM "notifications" WHERE "notifications"."is_active" = $1  ORDER BY "notifications"."id" DESC LIMIT 1  [["is_active", "t"]]
  User Load (0.4ms)  SELECT  "users".* FROM "users" WHERE "users"."id" = $1 LIMIT 1  [["id", 1]]
  Spree::Store Load (0.5ms)  SELECT  "spree_stores".* FROM "spree_stores"  ORDER BY "spree_stores"."id" ASC LIMIT 1
  Spree::Order Load (0.7ms)  SELECT  "spree_orders".* FROM "spree_orders" WHERE "spree_orders"."completed_at" IS NULL AND "spree_orders"."currency" = $1 AND "spree_orders"."guest_token" = $2 AND "spree_orders"."store_id" = 1 LIMIT 1  [["currency", "GBP"], ["guest_token", "O8yKUt6xgv-nCV4c6VmjHg1502111072660"]]
  Spree::Order Load (0.4ms)  SELECT  "spree_orders".* FROM "spree_orders" WHERE "spree_orders"."user_id" = $1 AND "spree_orders"."completed_at" IS NULL  ORDER BY created_at DESC LIMIT 1  [["user_id", 1]]
  Spree::LineItem Load (0.5ms)  SELECT "spree_line_items".* FROM "spree_line_items" WHERE "spree_line_items"."order_id" IN (24)  ORDER BY "spree_line_items"."created_at" ASC
  Spree::Variant Load (0.7ms)  SELECT "spree_variants".* FROM "spree_variants" WHERE "spree_variants"."deleted_at" IS NULL AND "spree_variants"."id" IN (40)
  Spree::Image Load (0.5ms)  SELECT "spree_assets".* FROM "spree_assets" WHERE "spree_assets"."type" IN ('Spree::Image') AND "spree_assets"."viewable_type" = 'Spree::Variant' AND "spree_assets"."viewable_id" IN (40)  ORDER BY "spree_assets"."position" ASC
  Spree::OptionValueVariant Load (0.3ms)  SELECT "spree_option_value_variants".* FROM "spree_option_value_variants" WHERE "spree_option_value_variants"."variant_id" IN (40)
  Spree::OptionValue Load (0.3ms)  SELECT "spree_option_values".* FROM "spree_option_values" WHERE "spree_option_values"."id" IN (7)
  Spree::Product Load (0.5ms)  SELECT "spree_products".* FROM "spree_products" WHERE "spree_products"."deleted_at" IS NULL AND "spree_products"."id" IN (28)
  Spree::Order Load (0.3ms)  SELECT "spree_orders".* FROM "spree_orders" WHERE "spree_orders"."user_id" = $1 AND "spree_orders"."completed_at" IS NULL AND (id != 24)  [["user_id", 1]]
  Rendered /home/kim/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/mountain_view-0.12.1/app/views/mountain_view/styleguide/index.html.erb within layouts/mountain_view (1.4ms)
Completed 500 Internal Server Error in 1607ms (ActiveRecord: 17.6ms)

NameError - undefined local variable or method `extra_pages' for #<#<Class:0x0055d5763a1ff0>:0x0055d576392fc8>
Did you mean?  extract_dimensions:
  spree_core (3.1.7) app/helpers/spree/base_helper.rb:71:in `method_missing'
  mountain_view (0.12.1) app/views/layouts/mountain_view.html.erb:16:in `_dfb447accbc97551ec927a44d6c75726'
  actionview (4.2.9) lib/action_view/template.rb:145:in `block in render'
  activesupport (4.2.9) lib/active_support/notifications.rb:166:in `instrument'
  actionview (4.2.9) lib/action_view/template.rb:333:in `instrument'
  actionview (4.2.9) lib/action_view/template.rb:143:in `render'
  deface (1.2.0) lib/deface/action_view_extensions.rb:41:in `render'
  actionview (4.2.9) lib/action_view/renderer/template_renderer.rb:66:in `render_with_layout'
  actionview (4.2.9) lib/action_view/renderer/template_renderer.rb:52:in `render_template'
  actionview (4.2.9) lib/action_view/renderer/template_renderer.rb:14:in `render'
  actionview (4.2.9) lib/action_view/renderer/renderer.rb:46:in `render_template'
  actionview (4.2.9) lib/action_view/renderer/renderer.rb:27:in `render'
  actionview (4.2.9) lib/action_view/rendering.rb:100:in `_render_template'
  actionpack (4.2.9) lib/action_controller/metal/streaming.rb:217:in `_render_template'
  actionview (4.2.9) lib/action_view/rendering.rb:83:in `render_to_body'
  actionpack (4.2.9) lib/action_controller/metal/rendering.rb:32:in `render_to_body'
  actionpack (4.2.9) lib/action_controller/metal/renderers.rb:37:in `render_to_body'
  actionpack (4.2.9) lib/abstract_controller/rendering.rb:25:in `render'
  actionpack (4.2.9) lib/action_controller/metal/rendering.rb:16:in `render'
  actionpack (4.2.9) lib/action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render'
  activesupport (4.2.9) lib/active_support/core_ext/benchmark.rb:12:in `block in ms'
  /home/kim/.rbenv/versions/2.4.1/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
  activesupport (4.2.9) lib/active_support/core_ext/benchmark.rb:12:in `ms'
  actionpack (4.2.9) lib/action_controller/metal/instrumentation.rb:44:in `block in render'
  actionpack (4.2.9) lib/action_controller/metal/instrumentation.rb:87:in `cleanup_view_runtime'
  activerecord (4.2.9) lib/active_record/railties/controller_runtime.rb:25:in `cleanup_view_runtime'
  actionpack (4.2.9) lib/action_controller/metal/instrumentation.rb:43:in `render'
  actionpack (4.2.9) lib/action_controller/metal/implicit_render.rb:10:in `default_render'
  actionpack (4.2.9) lib/action_controller/metal/implicit_render.rb:4:in `send_action'
  actionpack (4.2.9) lib/abstract_controller/base.rb:198:in `process_action'
  actionpack (4.2.9) lib/action_controller/metal/rendering.rb:10:in `process_action'
  actionpack (4.2.9) lib/abstract_controller/callbacks.rb:20:in `block in process_action'
  activesupport (4.2.9) lib/active_support/callbacks.rb:117:in `call'
  activesupport (4.2.9) lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
  activesupport (4.2.9) lib/active_support/callbacks.rb:505:in `call'
  activesupport (4.2.9) lib/active_support/callbacks.rb:92:in `__run_callbacks__'
  activesupport (4.2.9) lib/active_support/callbacks.rb:778:in `_run_process_action_callbacks'
  activesupport (4.2.9) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (4.2.9) lib/abstract_controller/callbacks.rb:19:in `process_action'
  actionpack (4.2.9) lib/action_controller/metal/rescue.rb:29:in `process_action'
  actionpack (4.2.9) lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
  activesupport (4.2.9) lib/active_support/notifications.rb:164:in `block in instrument'
  activesupport (4.2.9) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activesupport (4.2.9) lib/active_support/notifications.rb:164:in `instrument'
  actionpack (4.2.9) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
  actionpack (4.2.9) lib/action_controller/metal/params_wrapper.rb:250:in `process_action'
  activerecord (4.2.9) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
  actionpack (4.2.9) lib/abstract_controller/base.rb:137:in `process'
  actionview (4.2.9) lib/action_view/rendering.rb:30:in `process'
  actionpack (4.2.9) lib/action_controller/metal.rb:196:in `dispatch'
  actionpack (4.2.9) lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
  actionpack (4.2.9) lib/action_controller/metal.rb:237:in `block in action'
  actionpack (4.2.9) lib/action_dispatch/routing/route_set.rb:74:in `dispatch'
  actionpack (4.2.9) lib/action_dispatch/routing/route_set.rb:43:in `serve'
  actionpack (4.2.9) lib/action_dispatch/journey/router.rb:43:in `block in serve'
  actionpack (4.2.9) lib/action_dispatch/journey/router.rb:30:in `serve'
  actionpack (4.2.9) lib/action_dispatch/routing/route_set.rb:817:in `call'
  railties (4.2.9) lib/rails/engine.rb:518:in `call'
  railties (4.2.9) lib/rails/railtie.rb:194:in `method_missing'
  actionpack (4.2.9) lib/action_dispatch/routing/mapper.rb:51:in `serve'
  actionpack (4.2.9) lib/action_dispatch/journey/router.rb:43:in `block in serve'
  actionpack (4.2.9) lib/action_dispatch/journey/router.rb:30:in `serve'
  actionpack (4.2.9) lib/action_dispatch/routing/route_set.rb:817:in `call'
  rack (1.6.8) lib/rack/etag.rb:24:in `call'
  rack (1.6.8) lib/rack/conditionalget.rb:25:in `call'
  rack (1.6.8) lib/rack/head.rb:13:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/flash.rb:260:in `call'
  rack (1.6.8) lib/rack/session/abstract/id.rb:225:in `context'
  rack (1.6.8) lib/rack/session/abstract/id.rb:220:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/cookies.rb:560:in `call'
  activerecord (4.2.9) lib/active_record/query_cache.rb:36:in `call'
  activerecord (4.2.9) lib/active_record/connection_adapters/abstract/connection_pool.rb:653:in `call'
  activerecord (4.2.9) lib/active_record/migration.rb:377:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.2.9) lib/active_support/callbacks.rb:88:in `__run_callbacks__'
  activesupport (4.2.9) lib/active_support/callbacks.rb:778:in `_run_call_callbacks'
  activesupport (4.2.9) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (4.2.9) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/reloader.rb:73:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/remote_ip.rb:78:in `call'
  better_errors (2.3.0) lib/better_errors/middleware.rb:84:in `protected_app_call'
  better_errors (2.3.0) lib/better_errors/middleware.rb:79:in `better_errors_call'
  better_errors (2.3.0) lib/better_errors/middleware.rb:57:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  web-console (2.3.0) lib/web_console/middleware.rb:28:in `block in call'
  web-console (2.3.0) lib/web_console/middleware.rb:18:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.2.9) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.2.9) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.2.9) lib/active_support/tagged_logging.rb:68:in `block in tagged'
  activesupport (4.2.9) lib/active_support/tagged_logging.rb:26:in `tagged'
  activesupport (4.2.9) lib/active_support/tagged_logging.rb:68:in `tagged'
  railties (4.2.9) lib/rails/rack/logger.rb:20:in `call'
  quiet_assets (1.1.0) lib/quiet_assets.rb:27:in `call_with_quiet_assets'
  actionpack (4.2.9) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.6.8) lib/rack/methodoverride.rb:22:in `call'
  rack (1.6.8) lib/rack/runtime.rb:18:in `call'
  activesupport (4.2.9) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
  rack (1.6.8) lib/rack/lock.rb:17:in `call'
  actionpack (4.2.9) lib/action_dispatch/middleware/static.rb:120:in `call'
  rack (1.6.8) lib/rack/sendfile.rb:113:in `call'
  railties (4.2.9) lib/rails/engine.rb:518:in `call'
  railties (4.2.9) lib/rails/application.rb:165:in `call'
  rack (1.6.8) lib/rack/lock.rb:17:in `call'
  rack (1.6.8) lib/rack/content_length.rb:15:in `call'
  rack (1.6.8) lib/rack/handler/webrick.rb:88:in `service'
  /home/kim/.rbenv/versions/2.4.1/lib/ruby/2.4.0/webrick/httpserver.rb:140:in `service'
  /home/kim/.rbenv/versions/2.4.1/lib/ruby/2.4.0/webrick/httpserver.rb:96:in `run'
  /home/kim/.rbenv/versions/2.4.1/lib/ruby/2.4.0/webrick/server.rb:290:in `block in start_thread'
```

Thanks for any tips!

**Comment by kitop** [comment]
Mhm... taking a look at this.
Will be realeasing a new patch version soon!

**Comment by kitop** [comment]
@kimadactyl just released 0.12.2.

Can you try upgrading? Thanks!

**Comment by kimadactyl** [comment]
Wow, that did the trick, fastest bug fix ever, thankyou :100:
